app/controllers/api/v2/registration_controller.rb

```diff
@@ -92,7 +92,9 @@ def host
           return
         end
 
-        @host.setBuild
+        # Do not rebuild managed hosts
+        # https://bugzilla.redhat.com/show_bug.cgi?id=2212630
+        @host.setBuild unless @host.managed?
```

Comment:
I'm trying to understand this. IIRC an unmanaged host can't be in build mode (`setBuild` is only defined in `app/models/host/managed.rb`), so when is this ever doing something?

Reply:
The host registered with subscription-manager is created as a `Host::Managed`, but has `@host.managed? => false`. 
So that's why it works.

The `unless @host.managed?` condition is for hosts provisioned from the Foreman.

Reply:
Thanks for that explanation. I think that could be useful as a code comment.

Reply:
Yop, will add it